snuba/schemas.py

```diff
@@ -238,10 +207,48 @@
             ],
             'minItems': 3,
             'maxItems': 3,
-        }
+        },
     }
 }
 
+EVENTS_QUERY_SCHEMA = {
+    **copy.deepcopy(GENERIC_QUERY_SCHEMA),
```

Comment:
Eventually, I'd like to make this act more like `ChainMap` so that there is less deep copying, but that seems like an optimization that doesn't need to be made right now.